src/utils/authentication/utils/getUrlParameter.js

```diff
@@ -0,0 +1,8 @@
+export default (key) => {
+  const cleanKey = key.replace(/[\[]/, '\\[').replace(/[\]]/, '\\]')
+  const regex = new RegExp(`[\\?&]${cleanKey}=([^&#]*)`)
```

Comment:
I know inline comments aren't y'alls favorite, but I think they're pretty useful for regex :man_shrugging: